<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugFilterClass</class>
 <widget class="QWidget" name="DebugFilterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DebugFilter</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="cboSearch">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pBtnSearch">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pBtnReset">
     <property name="maximumSize">
      <size>
       <width>30</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="DebuggerUI.qrc">
       <normaloff>:/DebuggerUI/resources/remove.png</normaloff>:/DebuggerUI/resources/remove.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DebuggerUI.qrc"/>
 </resources>
 <connections/>
</ui>
